test/contextmanagers.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Context managers for using with IBMQProvider unit tests."""
+
+import os
+from contextlib import contextmanager
+
+
+@contextmanager
+def custom_envs(new_environ):
+    """Context manager that modifies environment variables."""
```

Comment:
Even if not present in the original file, can you add a docstring for the `new_environ` parameter, while we are at it?